src/features/sign-up/ui/SignUpForm.tsx

```diff
@@ -1,18 +1,23 @@
+import { FC } from 'react';
+
 import { zodResolver } from '@hookform/resolvers/zod';
 import { useForm, FormProvider, SubmitHandler } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
 
-import { VStack, Button, Text, Pressable, HStack } from '@shared/ui';
+import colors from '@app/shared/lib/constants/colors';
```

Comment:
Please remove this line as we can pass color variables out-of-the-box for Tamagui-like components